front/SettingsForm.tsx

```diff
@@ -72,6 +72,7 @@ const SettingsForm = () => {
               className="form-control form-control-lg"
               type="text"
               placeholder="Username"
+              data-cy="username-settings"
```

Comment:
Consider using a more descriptive name for the data-cy attribute, such as 'username-input' instead of 'username-settings'.